packages/extensions/openapi-to-typespec/src/transforms/transform-arm-resources.ts

```diff
@@ -704,8 +711,24 @@ function getTspOperations(armSchema: ArmResourceSchema): [TspArmResourceOperatio
   return [tspOperations, normalOperations];
 }
 
-function getOperationName(name: string): string {
-  return _.lowerFirst(_.last(name.split("_")));
+const existingNames: { [resourceName: string]: Set<string> } = {};
+// TO-DO: Figure out a way to create a new name if the name exists
```

Comment:
I think we want to align the logic with .NET SDK generator

Reply:
Yes, let's see later.